dot/state/service.go

```diff
@@ -43,6 +43,14 @@ type Service struct {
 	BabeThresholdDenominator uint64
 }
 
+// Pause Pauses the state service
+func (s *Service) Pause() error {
+	if !s.Block.IsPaused() {
+		return s.Block.Pause()
+	}
+	return nil
```

Comment:
Since `s.Block.Pause()` already has the `s.Block.IsPaused()` check we can simplify this logic to:

```suggestion
	return s.Block.Pause()
```

Reply:
nice call!